404.css

```diff
@@ -1,25 +1,46 @@
-.page_404{ padding:40px 0; background:#fff; font-family: 'Arvo', serif;
+*{
+    box-sizing: border-box;
+    margin: 0;
+    padding: 0;
 }
-
-.page_404  img{ width:100%;}
-
-.four_zero_four_bg{
-	background-image: url(https://cdn.dribbble.com/users/285475/screenshots/2083086/dribbble_1.gif);
-    height: 400px;
-    background-position: center;
+body{
```

Comment:
Donot make changes to body, make changes only to classnames. or id

Reply:
Should I merge the previous changes of 404 page(animated one) made by someone else?